tools/render-test/render-d3d11.cpp

```diff
@@ -455,10 +455,18 @@ class D3D11Renderer : public Renderer, public ShaderCompiler
 		ID3D11Buffer * buffer = nullptr;
 	};
 
+    UInt RoundUpToAlignment(UInt size, UInt alignment)
+    {
+        if (size % alignment)
+            return (size / alignment + 1) * alignment;
+        else
```

Comment:
Can’t this just be `((size + alignment - 1) / alignment) * alignment`?

Reply:
You are right, that is better.